provider/gateway/rest/router_test.go

```diff
@@ -307,6 +308,44 @@ func TestRouteDoesNotExist(t *testing.T) {
 	})
 }
 
+func TestRouteVersionOK(t *testing.T) {
+	runRouterTest(t, false, func(test *routerTest) {
+		status := versionInfo{
+			Akash: "",
```

Comment:
Please use a non-empty string here for the test. Anything is fine, just `"akashtest"` or whatever should be OK

Reply:
Done